DOCUMENTATION.md

```diff
@@ -0,0 +1,116 @@
+# Documentation
+
+## Flatseal permissions
+
+This section indicates the permissions of Flatpak with better integration with Flatseal. It is based on the official [Sandbox Permissions Reference](https://docs.flatpak.org/en/latest/sandbox-permissions-reference.html).
+
+Flatseal uses the `flatpak override` command. Every change the user does will make Flatseal run the `flatpak override` command to override the default permissions that the Flatpak program originally shipped with. All overridden permissions are located inside the `~/.local/share/flatpak/overrides` and `/var/lib/flatpak/overrides` directories. If you want to read more into `flatpak override`, you can look at the [`FLATPAK OVERRIDE(1)` manpage](https://manpages.debian.org/testing/flatpak/flatpak-override.1.en.html).
```

Comment:
Both `flatseal` and `flatpak-override` command line tool use the same overrides back-end (they both create override files).

That is the reason why both share the same options.

Reply:
Are you saying that what I wrote was false and suggesting to change it or are you explaining it further?

Reply:
> Are you saying that what I wrote was false and suggesting to change it or are you explaining it further?

My bad, I wasn't clear.

Flatseal doesn't use `flatpak-override` command line tool, instead, Flatseal produces the same output (override files) as the command line tool.

So that second paragraph should be corrected :)

Reply:
Is this good? https://github.com/TheEvilSkeleton/Flatseal/commit/1663d63e51f9addc6f6bc052f20877f90738ec09

Reply:
Looks correct!

Reply:
Feel free to resolve the conversation